markwon/src/main/java/ru/noties/markwon/renderer/SpannableMarkdownVisitor.java

```diff
@@ -64,7 +65,7 @@
 
     public SpannableMarkdownVisitor(
             @NonNull SpannableConfiguration configuration,
-            @NonNull SpannableBuilder builder
+            @NonNull SpannableStringBuilder builder
```

Comment:
Couldn't we just rename `ReverseSpannableStringBuilder` to `SpannableBuilder` and keep passing that around instead of accepting the super-class `SpannableStringBuilder` as parameter to the constructor. Here we could also add the handing of `nullable` and `array` spans.